pkgs/development/compilers/elm/packages/elm-review/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
```

Comment:
Run nixfmt.

Reply:
The new version, right?

Reply:
Yeah its avilable in `nix-shell` as `nixfmt 'path/to/file' `
You dont want to run it globally, only on your new/changed files